src/nvim/api/extmark.c

```diff
@@ -129,6 +134,34 @@ static Array extmark_to_array(ExtmarkInfo extmark, bool id, bool add_dict)
         ADD(chunks, ARRAY_OBJ(chunk));
       }
       PUT(dict, "virt_text", ARRAY_OBJ(chunks));
+      PUT(dict, "virt_text_hide", BOOLEAN_OBJ(decor->virt_text_hide));
+      PUT(dict, "virt_text_win_col", INTEGER_OBJ(decor->col));
```

Comment:
this should only be included if `decor->virt_text_pos == kVTWinCol`

Reply:
Thank you!
fixed.